skglm/solvers/cd_solver.py

```diff
@@ -417,11 +417,11 @@ def _cd_epoch_sparse(X_data, X_indptr, X_indices, y, w, Xw, datafit, penalty, fe
     penalty : Penalty
         Penalty.
 
-    feats : array, shape (n_features,)
-        The range of features.
+    ws : array, shape (n_features,)
```

Comment:
is the shape correct ?

Reply:
I hesitated between `n_features` and `ws_size`. I change it to `ws_size`.

Reply:
n_features is really the number of columns of X so yes, ws_size here is correct